### DISCUSSION TITLE
Unable to resolve action `JamesIves/github-pages-deploy-action@v4.4.3`, repository not found

### DISCUSSION THREAD

**Discussion by Nikitomas** [JamesIves/github-pages-deploy-action#1435] [Q&A] [answered]
Hello, 
While executing my workflows, i suddenly encountered the following error:
![image](https://github.com/JamesIves/github-pages-deploy-action/assets/2642595/d2f4d981-eaeb-4c05-9502-7927fc74f318)

I updated to the newest action (@v4.4.3) but it's still giving me 
![image](https://github.com/JamesIves/github-pages-deploy-action/assets/2642595/94349ee5-16ee-44f5-beea-ae96171478ec)

This happened after multiple successful executions to the workflow and with no change to the code. 

Any ideas? 

Thanks

**Comment by JamesIves** [comment, marked as answer]
There's been no code changes in the past week. Perhaps try again? Could be a blip in GitHub's service API.
